static/app/views/dashboardsV2/widgetBuilder/utils.tsx

```diff
@@ -123,6 +123,14 @@ export function normalizeQueries({
         query.fields = fields.filter(field => !columns.includes(field));
       }
 
+      if (
```

Comment:
can we add a comment here just so we know why we reset orderby to empty

Reply:
https://github.com/getsentry/sentry/pull/33953/commits/a3b19f890a2ad1d2da284b3badfea3ea555d8641
Added! Let me know if it needs improvement